specs/.gitignore

```diff
@@ -1 +1,2 @@
 book
+theme
```

Comment:
@rootulp, the theme dir can be similarly .gitignored in the app's repo

Reply:
thanks! Created https://github.com/celestiaorg/celestia-app/issues/3151